app/assets/stylesheets/layout/page_specific/_money_navigator_questionnaire.scss

```diff
@@ -139,37 +140,101 @@
 		}
 
 		&[data-question-multiple] {
+
 			.button--no, 
 			.button--yes {
 				display: inline-block;
 				margin: 0 $default-gutter * 2 0 0;
-
-				@include respond-to($mq-s) {
-					margin-right: $default-gutter * 1.5; 
-				}
-
+				margin-right: 52px; 
+			
 				@include respond-to($mq-m) {
-					margin-right: $default-gutter; 
+					margin-right: 52px;
+				}	
+
+				/* Base for label styling */
+				[type="checkbox"]:not(:checked),
+				[type="checkbox"]:checked {
+					position: absolute;
+					left: -9999px;
+				}
+				[type="checkbox"]:not(:checked) + label,
+				[type="checkbox"]:checked + label {
+					position: relative;
+					padding-left: 1.95em;
+					cursor: pointer;
 				}
 
-				.response__text {
+				/* checkbox aspect */
+				[type="checkbox"]:not(:checked) + label:before,
+				[type="checkbox"]:checked + label:before {
+					content: '';
+					position: absolute;
+					left: 0; top: 0;
 					@extend .button; 
 					padding: 0; 
-					text-align: left; 
+					text-align: left;
+				}
+
+				/* checked mark aspect */
+				[type="checkbox"]:not(:checked) + label:after,
+				[type="checkbox"]:checked + label:after {
+					content:'';
+					position: absolute;
+					top: 0em; left: 0;
+					font-size: 1.125rem;
+					font-weight: 300;
+					padding: 12px 22px;
+					border-radius: 4px;
 				}
 
-				.response__control {
-					@extend .visually-hidden; 
+				// &:checked + .response__text 
+				[type="checkbox"]:checked + label:after {
+					background: $color-green-secondary; 
+					color: $color-white; 
+					
+				}
 
-					&:checked + .response__text {
-						color: $color-white; 
-						background: $color-green-secondary; 
-					}
+				/* span within label */ 
+				span {
+					display:none;
+					
+				}
+			
+				/* accessibility */
+				[type="checkbox"]:checked:focus + label:before,
+				[type="checkbox"]:not(:checked):focus + label:before {
+					outline: solid 0.1875rem $color-input-focus-border-color;
 				}
+
 			}
 
-			.button--yes {
-				margin-right: 0;
+			.button--yes{
+				[type="checkbox"]:not(:checked) + label:before,
+				[type="checkbox"]:checked + label:before {
+					content:'Yes';
```

Comment:
The problem with using generated content for this is that we lose the translations. It would be possible to do this by making the content conditional on the `theme-cy` class that is attached in the `<body>` tag but really this content should be generated from the translation file.

Reply:
Ah yes, I haven't worked with translation files @davidtrussler.

Reply:
Happy for @davidtrussler to use his suggested approach as long as input is not hidden and is keyboard/screen reader navigable.

Reply:
I've reworked the buttons so that they use the translation <span> rather than the Yes/No coming from the CSS. totally my bad on that!

Reply:
& my git command line captures spelling mistakes then uses them later. I've cleared the cache but sorry for the comments!